app/export_test.go

```diff
@@ -284,7 +284,7 @@ func TestExportDMChannelToSelf(t *testing.T) {
 	// Ensure no channels were imported
 	channels, err = th2.App.Srv.Store.Channel().GetAllDirectChannelsForExportAfter(1000, "00000000")
 	require.Nil(t, err)
-	assert.Equal(t, 0, len(channels))
+	assert.Equal(t, 1, len(channels))
```

Comment:
Could we check for something else here, like name or members of the newly created channel?